docs/readmes/orc8r/dev_gateway_registration.md

```diff
@@ -0,0 +1,85 @@
+---
+id: dev_gateway_registration
+title: Gateway Registration
+hide_title: true
+---
+# Gateway Registration
+
+## Motivation
+
+Previously, to register a gateway, the operator would have to gather the gateway information (both the hardware id and the challenge key) through `show_gateway_info.py` script, then provision the gateway with the correct values of `control_proxy.yml` and `rootCA.pem`.
+Then, the operator would have to register each gateway through the NMS UI and validate the gateway registration by running `checkin_cli.py`.  
+
+In efforts to simplify gateway registration, the new registration process only requires two steps: registering the gateway at Orc8r and then running `register.py` at the gateway.  
+
+## Overview
+
+The overview of gateway registration is as follows:
+![gateway_registration_overview](assets/orc8r/gateway_registration_overview.png)
+
+Note: The operator should set per-tenant default `control_proxy.yml` through the API endpoint `\tenants\{tenant_id}\control_proxy` as a prerequisite.
+The control proxy must have `\n` characters as line breaks. Here is a sample request body: 
+```json
+{
+   control_proxy: "nghttpx_config_location: /var/tmp/nghttpx.conf\n\nrootca_cert: /var/opt/magma/certs/rootCA.pem\ngateway_cert: /var/opt/magma/certs/gateway.crt\ngateway_key: /var/opt/magma/certs/gateway.key\nlocal_port: 8443\ncloud_address: controller.magma.test\ncloud_port: 7443\n\nbootstrap_address: bootstrapper-controller.magma.test\nbootstrap_port: 7444\nproxy_cloud_connections: True\nallow_http_proxy: True"
+}
+```
+
+1. The operator registers a gateway incompletely (i.e. without its device field) through Orc8r through an API call. It will receive a registration token, the `rootCA.pem`, and its Orc8r's domain name in the response body under the field registration info. For example,
```

Comment:
```suggestion
1. The operator registers a gateway partially (i.e. without its device field) through Orc8r through an API call. It will receive a registration token, the `rootCA.pem`, and its Orc8r's domain name in the response body under the field registration info. For example,
```
"Incompletely" sounds to me, as if the operator did something wrong, however, this is the desired new workflow, I believe.